<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addUI</class>
 <widget class="QDialog" name="addUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowOpacity">
   <double>0.850000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">#addUI
{
	background:rgba(12,10,20,0.3);
}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton
{
	background:rgba(246, 249, 245,0.15);
	font: 87 10pt &quot;Arial Black&quot;;
	color:  rgb(215, 223, 218);
}
</string>
   </property>
   <property name="text">
    <string>确认添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_2
{
	background:rgba(246, 249, 245,0.15);
	font: 87 10pt &quot;Arial Black&quot;;
	color:  rgb(215, 223, 218);
}
</string>
   </property>
   <property name="text">
    <string>完成添加</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>72</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label
{
	background:rgba(12,10,20,0.3);
	font: 87 14pt &quot;Arial Black&quot;;
	color:  rgb(215, 223, 218);
}
</string>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>132</y>
     <width>50</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_2
{
	background:rgba(12,10,20,0.3);
	font: 87 14pt &quot;Arial Black&quot;;
	color:  rgb(215, 223, 218);
}
</string>
   </property>
   <property name="text">
    <string>电话</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_3
{
	background:rgba(246, 249, 245,0.15);
	font: 87 10pt &quot;Arial Black&quot;;
	color:  rgb(215, 223, 218);
}
</string>
   </property>
   <property name="text">
    <string>一键导入</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>14</pointsize>
     <weight>10</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_3
{
	background:rgba(12,10,20,0.3);
	font: 87 14pt &quot;Arial Black&quot;;
	color:  rgb(215, 223, 218);
}
</string>
   </property>
   <property name="text">
    <string>地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>14</pointsize>
     <weight>10</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_4
{
	background:rgba(12,10,20,0.3);
	font: 87 14pt &quot;Arial Black&quot;;
	color:  rgb(215, 223, 218);
}
</string>
   </property>
   <property name="text">
    <string>关系</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
